factory/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 3.2.0
 
+* Updated default node version from `20.8.1` to `20.9.0`. Addressed in [#987](https://github.com/cypress-io/cypress-docker-images/pull/987)
```

Comment:
Is there a pull request 952? When I click this it takes me to a seemingly unrelated issue

Reply:
Yeah, I already updated it to [987](https://github.com/cypress-io/cypress-docker-images/pull/987)